### PR TITLE
MGMT-11733: added e2e-vsphere-assisted-kube-api-periodic job

### PR DESCRIPTION
**Pull request by danielerez**
Added a weekly job for vSphere kube-api test.

### CODE REVIEW

**Review by osherdp — CHANGES_REQUESTED** [review]
looks good but you'll need ``make jobs``

**Comment by danielerez** [comment]
> looks good but you'll need `make jobs`

right, added.

File: ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml [hunk 1]
```diff
@@ -1500,6 +1500,76 @@ periodics:
...
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: assisted-test-infra
+  labels:
+    ci-operator.openshift.io/cloud: vsphere
+    ci-operator.openshift.io/cloud-cluster-profile: vsphere
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-kube-api-periodic
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml:1516]: We're missing the slack notification here

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml (hunk 1 above)
Comment (danielerez) [reply ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml:1516]: shouldn't 'make jobs' add it?

Comment (osherdp) [reply ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml:1516]: nope

Comment (danielerez) [reply ci-operator/jobs/openshift/assisted-test-infra/openshift-assisted-test-infra-master-periodics.yaml:1516]: ack, added.

**Review by osherdp — APPROVED** [review]

**Pull request merged**